koyo-lib/koyo/url.rkt

```diff
@@ -66,5 +67,12 @@
   (make-parameter (lambda (name . args)
                     (error "current-reverse-uri-fn not installed"))))
 
-(define (reverse-uri . args)
-  (apply (current-reverse-uri-fn) args))
+(define/contract (reverse-uri #:query [params null] . args)
+  (->* ()
+       (#:params (listof (cons symbol? (or/c false/c string?))))
```

Comment:
`#:params` should be `#:query` here. Also I think it makes sense to extract the "where" parameter and make it required:

```racket
(define/contract (reverse-uri where #:query [params null] . args)
  (->* (symbol?)
       (#:query ...)
       #:rest any/c 
       string?)
  (define uri (apply (current-reverse-uri-fn) where args)
  ...)
```

Reply:
you're right regarding `#:params` and `#:query` 😅 I'd actually fixed that but tried to push to the wrong remote so it got rejected without me noticing :(

Also makes sense to make the "where" required!